kura/org.eclipse.kura.wire.component.provider/src/main/java/org/eclipse/kura/internal/wire/publisher/CloudPublisher.java

```diff
@@ -81,7 +81,7 @@ public final class CloudPublisher implements WireReceiver, ConfigurableComponent
      * Binds the Wire Helper Service.
      *
      * @param wireHelperService
-     *            the new Wire Helper Service
+     *                              the new Wire Helper Service
```

Comment:
What is happening here? Use the kura formatter and cleanup profiles if using Eclipse (https://github.com/eclipse/kura/tree/develop/kura/setups/formatting). Replace tabs with spaces.

Reply:
You're right, I followed this [documentation page](https://wiki.eclipse.org/Kura/Getting_Started) and was the cleanup/formatter v2 and not the updated v3.